exchange/gdpr.go

```diff
@@ -0,0 +1,92 @@
+package exchange
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+// ExtractGDPR will pull the gdpr info from an openrtb request
+func extractGDPR(bidRequest *openrtb.BidRequest, usersyncIfAmbiguous bool) (gdpr int, consent string, err error) {
```

Comment:
small note... it's a little weird that this is one function call. Parsing `gdpr` and parsing `consent` are two disjoint tasks.

Two smaller functions are generally better than one big one. More flexible, reusable, and easy to test.